### DISCUSSION TITLE
Can't log in, and nothing else charges

### DISCUSSION THREAD

**Discussion by hola1u1** [ppy/osu#17858] [Q&A]
### Type

Game behaviour

### Bug description

The game starts fine, but I can't log in my account and nothing else charges as if my phone didn't have any internet connection.
At last, after closing the game the music continues on second plane.

### Screenshots or videos

https://drive.google.com/drive/folders/188EVJZ6YX1pMrH1SCJ-1bLi_wdJIr4A4

### Version

2022.418.0-lazer

### Logs

[Google Drive osu logs](https://drive.google.com/drive/folders/1841bryyIa5KdJgW-iGPKAb-bwLLhRijS)

**Comment by frenzibyte** [comment]
Can't access logs.

<img width="402" alt="CleanShot 2022-04-18 at 07 51 20@2x" src="https://user-images.githubusercontent.com/22781491/163756405-2a1a54dd-58e8-48c1-a0e4-46d811632b57.png">

**Reply by hola1u1** [reply]
Now you can see them, sorry

**Comment by bdach** [comment]
logs contain this:

> Request to https://osu.ppy.sh/oauth/token failed with System.Net.WebException: Request to https://osu.ppy.sh/oauth/token timed out after 62 seconds idle (read 0 bytes, retried 0 times). (retrying 1/1).

which suggests an unspecified connection issue on your end. can you access the website correctly?
